src/annotator/adder.js

```diff
@@ -193,6 +189,50 @@ export class Adder {
     return { top, left, arrowDirection };
   }
 
+  /**
+   * Find the zIndex of five nearby elements (left-top, left-bottom, middle-center,
+   * right-top, right-bottom) and use the greatest.
+   *
+   * @param {number} left - Horizontal offset from left edge of viewport.
+   * @param {number} top - Vertical offset from top edge of viewport.
+   * @return {number} - greatest zIndex (default value of 1)
+   */
+  _findZindex(left, top) {
+    if (document.elementsFromPoint === undefined) {
+      // In case of not be able to find the zIndex of the surrounding elements,
+      // default to the greatest <integer> allowed in Opera (2^15).
+      // The <integer> value is currently enforced in the specification,
+      // and Opera has one of the smaller ranges.
+      // https://developer.mozilla.org/en-US/docs/Web/CSS/integer#Syntax
+      return 32768;
```

Comment:
I'm pretty sure that information on MDN is out of date. See https://joeist.com/2012/06/what-is-the-highest-possible-z-index-value/ (note this is from 2012). The actual value chosen here (2^15) should be fine though, as it is similar to what was used before.

I suppose we could have just hard-coded the z-index to 2^31-1. The downside is that there is a small chance that this might interfere with legitimate tools that need to display above the adder.

Reply:
OK.

For curiosity, what other legitimate tools are you referring?

Reply:
I didn't have anything specific in mind, I'm just guessing that _maybe_ there are such tools.